### ISSUE TITLE
Issue about code

### ISSUE THREAD

**Issue by muzol132126** [liuliu3dv/GOC#1]
A good and insightful work! 
I would like to ask when the code will be open source.

**Comment by muzol132126** [comment]
Thank you for your update.
 Looking forward to the paper's acceptance and the subsequent open sourcing of the project.

**Comment by PannagaS** [comment]
Yes, I am eagerly waiting for the implementation too! Very cool paper btw @liuliu3dv  :) 

Thanks!
